src/read/line.rs

```diff
@@ -575,6 +574,49 @@ fn line_target_yaw(i: &str) -> IResult<VectorialStrafingConstraints> {
                 tolerance: tolerance.unwrap_or(0.),
             },
         ),
+        map(
+            preceded(
+                tag("look_at"),
+                preceded(
+                    tag(" "),
+                    alt((
+                        map(
+                            preceded(
+                                tag("entity"),
+                                preceded(
+                                    tag(" "),
+                                    tuple((
+                                        cut(non_zero_u32),
+                                        opt(tuple((
```

Comment:
I'm pretty sure you can construct this parser in such a way that this part is not duplicated. I.e.
```
tuple((
  opt(parse_entity_index),
  preceded(tag(" "), parse_coordinates)
))
```

Reply:
hmmm wowzer this is very nice. I should have thought of that. I wasn't sure how to make the make the coordinate conditional.

Reply:
hmmm, just thought about this a bit. So I guess XYZ coordinate is always required if we go with that format? It's fine by me. The way I did for the writer is now that it will always generate "0 0 0" for entity index when it is not specified. That's the reason I do `alt()` at first.

Reply:
Oh, hm, yeah then you'll need an alt, but still should be nice after extracting into functions.